android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -76,4 +77,39 @@ open class Configuration @JvmOverloads constructor(
     companion object {
         const val MIN_TIME_BETWEEN_SESSIONS_MILLIS: Long = 300000
     }
+
+    @Deprecated("Please use 'autocapture.sessions' instead.", ReplaceWith("autocapture.sessions"))
+    var trackingSessionEvents: Boolean
+        get() = AutocaptureOption.SESSIONS in autocapture
+        set(value) {
+            if (value) autocapture += AutocaptureOption.SESSIONS else autocapture -= AutocaptureOption.SESSIONS
+        }
+
+    @Suppress("DEPRECATION")
```

Comment:
Why do we suppress deprecation here?

Reply:
This was to suppress the warning that defaultTracking is deprecated. This is removed on the new logic which is much easier to maintain. Please see b4c09984d035a3b408719a2e06c3d42eb4c2c01a.